### PR TITLE
BUG: fix unicode handling in PKG-INFO and metadata summary.

### LINKED ISSUE #71: Unicode files written to the zip file must be encoded to bytes.
When repacking an egg with a unicode description (and therefore unicode characters in the `PKG-INFO`):

```python
Traceback (most recent call last):
  File ".devenv/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File ".devenv/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File ".devenv/lib/python2.7/site-packages/enstaller/tools/repack.py", line 250, in <module>
    main()
  File ".devenv/lib/python2.7/site-packages/enstaller/tools/repack.py", line 246, in main
    repack(ns.egg, ns.build_number, ns.platform_string)
  File ".devenv/lib/python2.7/site-packages/enstaller/tools/repack.py", line 218, in repack
    with EggBuilder(metadata, cwd=parent_dir) as target:
  File ".devenv/lib/python2.7/site-packages/okonomiyaki/file_formats/egg.py", line 35, in __init__
    self._write_pkg_info()
  File ".devenv/lib/python2.7/site-packages/okonomiyaki/file_formats/egg.py", line 108, in _write_pkg_info
    self._fp.writestr(_PKG_INFO_LOCATION, data)
  File ".devenv/lib/python2.7/site-packages/zipfile2/_zipfile.py", line 116, in writestr
    compress_type)
  File ".devenv/lib/python2.7/zipfile.py", line 1234, in writestr
    zinfo.CRC = crc32(bytes) & 0xffffffff       # CRC-32 checksum
UnicodeEncodeError: 'ascii' codec can't encode characters in position 268-269: ordinal not in range(128)

Fatal error: local() encountered an error (return code 1) while executing '.devenv/bin/python -m enstaller.tools.repack -b 2 -a osx-64 my_package-1.3.0.dev0-py2.7.egg'
```

### PR DESCRIPTION
**Pull request by cournape**
Fixes #71 

@sjagoe

### CODE REVIEW

**Comment by cournape** [comment]
Since Simon is on holidays: @itziakos

File: okonomiyaki/file_formats/egg.py [hunk 1]
```diff
@@ -93,14 +93,15 @@ def commit(self):
 
     def _write_spec_depend(self):
         spec_depend_string = self._egg_metadata.spec_depend_string
-        self._fp.writestr(_SPEC_DEPEND_LOCATION, spec_depend_string)
+        self._fp.writestr(_SPEC_DEPEND_LOCATION, spec_depend_string.encode())
```
Comment (jvkersch) [inline okonomiyaki/file_formats/egg.py]: Why not specify the encoding here as well?

**Comment by jvkersch** [comment]
LGTM

File: okonomiyaki/file_formats/egg.py (hunk 1 above)
Comment (cournape) [inline okonomiyaki/file_formats/egg.py]: it should indeed be ascii

**Pull request merged**